.github/workflows/lint-validate.yml

```
@@ -29,30 +29,50 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install yamllint
 
-      # We use --no-warnings in this step to reduce output to critical errors
-      - name: Run yamllint on config/
-        run: |
-          yamllint -d scripts/yamllint-config.yaml --no-warnings config/
-
       - name: Unlock git-crypt secrets
         if: github.event.pull_request.head.repo.fork == false
         uses: sliteteam/github-action-git-crypt-unlock@a09ea5079c1b0e1887d4c8d7a4b20f00b5c2d06b
         env:
           GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
 
+      # pick yamllint config based on whether we have access to secrets or not
+      - name: Select yamllint config
+        run: |
+          if [[ "${{ github.event.pull_request.head.repo.fork }}" == "false" ]]; then
+            echo "linting secrets"
+            export YAMLLINT_CONFIG=scripts/yamllint-config.yaml
+          else
+            echo "not linting secrets"
+            export YAMLLINT_CONFIG=scripts/yamllint-no-secrets.yaml
+          fi
+          # persist env for the next step
+          echo "::set-env name=YAMLLINT_CONFIG::$YAMLLINT_CONFIG"
+
       # We use --no-warnings in this step to reduce output to critical errors
-      - name: Run yamllint on secrets/config
-        if: github.event.pull_request.head.repo.fork == false
+      - name: Run yamllint
         run: |
-          yamllint -d scripts/yamllint-config.yaml --no-warnings secrets/config/
+          yamllint -d ${YAMLLINT_CONFIG} --no-warnings .
 
-  yamllint-templates:
-    if: github.event.pull_request.head.repo.fork == false
+  lint-helm:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        release: ["staging", "prod", "ovh", "turing"]
+        include:
+        # versions must be available
+        # at https://github.com/instrumenta/kubernetes-json-schema
+        # in the form v${version}-standalone-strict
+        # this does *not* include all kubernetes versions!
+        - release: staging
+          kube_version: "1.15.7"
+        - release: prod
+          kube_version: "1.15.7"
+        - release: prod
+          kube_version: "1.18.1"
+        - release: ovh
+          kube_version: "1.15.7"
+        - release: turing
+          kube_version: "1.16.4"
```

```suggestion
          kube_version: "1.16.10"
```

This is Turing's current `kubectl` version

---

I added a comment explaining the reason - this can't actually be the full version because not all version have published schemas. It has to be listed here: https://github.com/instrumenta/kubernetes-json-schema

---

Whoops, missed that! 😄